macOS/DuckDuckGo/AIChat/UserScript/AIChatMessageHandler.swift

```diff
@@ -0,0 +1,91 @@
+//
+//  AIChatMessageHandler.swift
+//
+//  Copyright © 2025 DuckDuckGo. All rights reserved.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//
+
+enum AIChatMessageType {
+    case nativeConfigValues
+    case nativeHandoffData
+    case nativePrompt
+}
+
+protocol AIChatMessageHandling {
```

Comment:
I'm planning to move this to a shared AIChat package with iOS. It will be done as part of the AOR, didn't want to scope creep the project now.